packages/core/src/components/Icon/Icons/components/Heart.tsx

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable */
+/* tslint:disable */
+import * as React from 'react';
+export interface HeartProps extends React.SVGAttributes<SVGElement> {
+size?: string | number;
+}
+const Heart: React.FC<HeartProps> = ({size, ...props}) => (
+  <svg viewBox="0 0 20 20" fill="currentColor" width={ size || "20" } height={ size || "20" } {...props}>
+    <path d="M4.64486 10.3388C4.64977 10.3436 4.65466 10.3484 4.65951 10.3532L9.99574 15.6895L15.3329 10.3523C15.3377 10.3475 15.3426 10.3427 15.3475 10.3379C15.3785 10.3086 15.3899 10.2982 15.3785 10.3086C15.3823 10.3051 15.3861 10.3016 15.3899 10.2982C16.0674 9.68825 16.4914 8.80691 16.4914 7.8257C16.4914 5.98897 15.0024 4.5 13.1657 4.5C12.1647 4.5 11.5513 5.06987 11.0596 5.55744L11.0519 5.56512C10.7694 5.84526 10.3871 6.00169 9.98928 5.99999C9.59143 5.99829 9.21055 5.8386 8.93045 5.55606C8.45718 5.07868 7.84112 4.5 6.8257 4.5C4.98897 4.5 3.5 5.98897 3.5 7.8257C3.5 8.80725 3.92427 9.68887 4.60218 10.2988C4.60601 10.3023 4.60982 10.3058 4.61362 10.3093L4.64486 10.3388ZM10.5261 17.2805L16.3935 11.413C17.3735 10.5307 17.9914 9.24988 17.9914 7.8257C17.9914 5.16054 15.8308 3 13.1657 3C11.5326 3 10.5394 3.96129 10.0258 4.47012C10.0183 4.47762 10.0108 4.48502 10.0034 4.49232L9.99569 4.5C9.99174 4.49601 9.98775 4.49199 9.98373 4.48794C9.484 3.98411 8.48664 3 6.8257 3C4.16054 3 2 5.16054 2 7.8257C2 9.25038 2.61832 10.5316 3.59885 11.4139L9.46541 17.2805C9.60606 17.4211 9.79683 17.5001 9.99574 17.5001C10.1947 17.5001 10.3854 17.4211 10.5261 17.2805Z"
+      fill="#676879" fillRule="evenodd" clipRule="evenodd" />
```

Comment:
The fill should be currentColor no?

Reply:
@rivka-ungar you should re-generate the icons to update this one as well

Reply:
oops